x-pack/plugins/security_solution/public/management/pages/mocks/trusted_apps_http_mocks.ts

```diff
@@ -148,6 +149,11 @@ export const trustedAppsGetOneHttpMocks =
         exceptionItem.namespace_type =
           apiQueryParams.namespace_type ?? exceptionItem.namespace_type;
 
+        // map process.hash entries to have * as suffix
```

Comment:
Any way we can do this outside of the HTTP mock?
Since this is supposed to mock the HTTP response, it would be best if no mappings were done here and that this can remain as close as possible to a real api response.

Reply:
so I've moved this to the [test](https://github.com/elastic/kibana/pull/129208/commits/fbaa8c7e221e63421177d729141268071c3b7449#diff-382d83e85dfee73b16266bd79febecb047a7f5f09733504fa7946435906fd4dcR362-R374) that needs this mapping.

Reply:
so I've moved this to the [test](https://github.com/elastic/kibana/pull/129208/commits/fbaa8c7e221e63421177d729141268071c3b7449#diff-382d83e85dfee73b16266bd79febecb047a7f5f09733504fa7946435906fd4dcR362-R374) that needs this mapping.